en/docs/reference/connectors/smpp-connector/smpp-inbound-endpoint-example.md

```diff
@@ -12,20 +12,47 @@ Following diagram shows the overall solution we are going to build. The SMSC wil
 
 <img src="{{base_path}}/assets/img/integrate/connectors/smpp-inboundep-example.png" title="SMPP Inbound Endpoint" width="800" alt="SMPP Inbound Endpoint"/>
 
-## Configure inbound endpoint using WSO2 Integration Studio
+## Setting up the inbound endpoint using Micro Integrator
```

Comment:
```suggestion
## Set up the inbound endpoint using Micro Integrator
```

Reply:
Since we have followed a similar convention of using the -ing form for all ## titles across the pages in the doc space, like 'Adding' and 'Setting Up' is it okay to keep this as 'Setting up'?